### ISSUE TITLE
The accuracy is very poor, am I doing something wrong?

### ISSUE THREAD

**Issue by tu1258** [google/qkeras#44]
**Code**

```
input_audio = Input(
                shape=flags.desired_samples,
                batch_size=flags.batch_size,
                dtype=tf.float32)
  net = input_audio

  frame_size = int(flags.sample_rate * flags.window_size_ms / 1000)
  frame_stride = int(flags.sample_rate * flags.window_stride_ms / 1000)
  net = dataframe.DataFrame(
        inference_batch_size=1,
        frame_size=frame_size,
        frame_step=frame_stride)(
        net)
  net = windowing.Windowing(
        window_size=frame_size, 
        window_type=flags.window_type)(
        net)
  net = magnitude_rdft_mel.MagnitudeRDFTmel(
        num_mel_bins=flags.mel_num_bins,
        lower_edge_hertz=flags.mel_lower_edge_hertz,
        upper_edge_hertz=flags.mel_upper_edge_hertz,
        sample_rate=flags.sample_rate,
        mel_non_zero_only=flags.mel_non_zero_only)(
        net)
  net = Lambda(lambda x: tf.math.log(tf.math.maximum(x, flags.log_epsilon)))(net)
  net = dct.DCT(num_features=flags.dct_num_features)(net)
          

  time_size, feature_size = net.shape[1:3]
  channels = utils.parse(flags.channels)
  net = tf.keras.backend.expand_dims(net)
    
  net = tf.reshape(
      net, [-1, time_size, 1, feature_size])  # [batch, time, 1, feature]
  first_kernel = utils.parse(flags.first_kernel)
  conv_kernel = utils.parse(flags.kernel_size)
  groups = flags.groups
  layer = 0

  net = qkeras.QConv2D(
        filters=channels[0],
        kernel_size=first_kernel,
        strides=1,
        padding='same',
        use_bias=True,
        activation='linear',
        kernel_quantizer='quantized_bits(8)',
        name='first_conv')(
          net)

  net = qkeras.QBatchNormalization(
        momentum=0.997,
        name='first_bn')(
          net)
  net = qkeras.QActivation('quantized_bits(8)',name='first_quantize')(net)
  net = Activation('relu',name='first_act')(net)

  channels = channels[1:]

  for n in channels:

    if(flags.mobile):
      net = qkeras.QDepthwiseConv2D(
            kernel_size=conv_kernel,
            strides=1,
            padding='same',
            use_bias=True,
            activation='linear',
            depthwise_quantizer='quantized_bits(8)',
            name='dw'+str(layer))(
              net)
      net = qkeras.QActivation('quantized_bits(8)',name='quantize_mobile'+str(layer))(net)
      net = qkeras.QConv2D(
            filters=n,
            # groups=groups,
            kernel_size=1,
            strides=1,
            padding='same',
            use_bias=True,
            activation='linear',
            kernel_quantizer='quantized_bits(8)',
            name='pw'+str(layer))(
              net)
        
    else:
      net = qkeras.QConv2D(
            filters=n,
            # groups=groups,
            kernel_size=conv_kernel,
            strides=1,
            padding='same',
            use_bias=True,
            activation='linear',
            kernel_quantizer='quantized_bits(8)',
            name='conv'+str(layer))(
              net)
      
    net = qkeras.QBatchNormalization(
          momentum=0.997,
          name='bn'+str(layer))(
            net)

    net = qkeras.QActivation('quantized_bits(8)',name='quantize'+str(layer))(net)
    net = Activation('relu',name='act'+str(layer))(net)
    layer = layer + 1

  net = AveragePooling2D(pool_size=net.shape[1:3], strides=1,name='pool')(net)

  # net = tf.keras.layers.Dropout(rate=flags.dropout)(net)

  # fully connected layer
  net = qkeras.QConv2D(
        filters=flags.label_count,
        kernel_size=1,
        strides=1,
        padding='same',
        use_bias=True,
        activation='linear',
        name='last_conv')(
          net)

  net = tf.reshape(net, shape=(-1, net.shape[3]), name='reshape')
  return tf.keras.Model(input_audio, net)
```

The first few layers are preprocessing layers and don't affect the quantization.
